src/core/unit.c

```diff
@@ -3633,16 +3635,50 @@ int unit_install_bus_match(Unit *u, sd_bus *bus, const char *name) {
         if (r < 0)
                 return r;
 
-        r = sd_bus_call_method_async(
-                        bus,
-                        &u->get_name_owner_slot,
-                        "org.freedesktop.DBus",
-                        "/org/freedesktop/DBus",
-                        "org.freedesktop.DBus",
-                        "GetNameOwner",
-                        get_name_owner_handler,
-                        u,
-                        "s", name);
+        r = sd_bus_message_new_method_call(bus,
+                                           &m1,
+                                           "org.freedesktop.DBus",
+                                           "/org/freedesktop/DBus",
+                                           "org.freedesktop.DBus",
+                                           "AddMatch");
```

Comment:
we'd usually line break/indent this like this:

```c
r = sd_bus_message_new_method_call(
            bus,
            &m1, 
            …
```

i.e. break after the `(`, and then do double our usual indentation, i.e. 16 spaces. please follow this here too